### PR TITLE
New framework: tokio-minihttp

### PR DESCRIPTION
**Pull request by gkbrk**

### CODE REVIEW

File: frameworks/Rust/tokio-minihttp/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+version = "0.1.0"
+authors = ["Gökberk Yaltıraklı <webdosusb@gmail.com>"]
+
+[dependencies]
+futures = "0.1"
+httparse = "1.1"
+serde_json = "0.8"
+num_cpus = "1.0"
+tokio-proto = { git = "https://github.com/tokio-rs/tokio-proto" }
+tokio-service = { git = "https://github.com/tokio-rs/tokio-service" }
```
Comment (nbrady-techempower) [inline frameworks/Rust/tokio-minihttp/Cargo.toml]: hi @gkbrk thanks for the submission!
If you're using git repos as a source they need to be tied down to specific commits or releases

Comment (gkbrk) [inline frameworks/Rust/tokio-minihttp/Cargo.toml]: @nbrady-techempower Working on it 👍

Comment (gkbrk) [inline frameworks/Rust/tokio-minihttp/Cargo.toml]: @nbrady-techempower Hopefully it should work now

**Comment by nbrady-techempower** [comment]
@gkbrk Thanks for updating. Will let travis do its thing and then test locally. Could you add a `README.md` with a little information about the framework when you have a chance?

**Comment by gkbrk** [comment]
@nbrady-techempower There's something very wrong with the Travis build. https://travis-ci.org/TechEmpower/FrameworkBenchmarks/builds/188637698

File: frameworks/Rust/tokio-minihttp/benchmark_config.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "classification": "Micro",
+      "database": "None",
+      "framework": "tokio-minihttp",
+      "language": "rust",
+      "orm": "raw",
+      "platform": "Rust",
+      "webserver": "tokio-minihttp",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "tokio-minihttp",
```
Comment (nbrady-techempower) [inline frameworks/Rust/tokio-minihttp/benchmark_config.json]: Trailing commas are not valid in current JSON spec. This is breaking your `benchmark_config`. You can see a note about your benchmark_config in every failing test.

**Comment by nbrady-techempower** [comment]
@gkbrk I took care of that for you, so we'll see the new run. It passes locally.

**Pull request merged**